docs/tutorials/03_ground_state_solvers.ipynb

```diff
@@ -22,18 +22,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/bpa/opt/anaconda3/envs/qiskit-nature/lib/python3.7/site-packages/pyscf/lib/misc.py:46: H5pyDeprecationWarning: Using default_file_mode other than 'r' is deprecated. Pass the mode to h5py.File() instead.\n",
+      "  h5py.get_config().default_file_mode = 'a'\n"
+     ]
+    }
+   ],
    "source": [
-    "from qiskit.chemistry.drivers import PySCFDriver, UnitsType, Molecule\n",
-    "from qiskit.chemistry.transformations import FermionicTransformation, FermionicQubitMappingType\n",
+    "from qiskit import Aer\n",
+    "from qiskit_nature.drivers import PySCFDriver, UnitsType, Molecule\n",
+    "from qiskit_nature.problems.second_quantization import ElectronicStructureProblem\n",
+    "from qiskit_nature.operators.second_quantization.qubit_converter import QubitConverter\n",
```

Comment:
I see here the ESP import is direct from 2nd Q whereas QConv is not (goes back to a prior comment on these imports)